source/options.tsx

```diff
@@ -99,13 +109,9 @@ async function updateStorageUsage(area: 'sync' | 'local'): Promise<void> {
 }
 
 async function validateToken(): Promise<void> {
-	reportStatus({});
 	const tokenField = select('input[name="personalToken"]')!;
-
-	if (tokenField.value.startsWith('github_pat_')) {
-		// Validation not supported yet https://github.com/refined-github/refined-github/issues/6092
-		return;
-	}
+	const tokenType = tokenField.value.startsWith('github_pat_') ? 'fine_grained' : 'classic';
+	reportStatus({tokenType});
```

Comment:
Just reporting after tokenType detecting.
If it reported duplicately, UI will being flickering.
Maybe we can drop undefinable attribute on `tokenType` of `reportStatus` param.